### PR TITLE
GLTFExporter: Add null check for skeleton while processing skinned mesh.

### PR DESCRIPTION
**Pull request by metipton**
I have an situation where I have skinned meshes without the attached skeletons on export and need to check if skeleton is null before proceeding

### CODE REVIEW

**Review by donmccurdy — APPROVED** [review]

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1648,7 +1648,7 @@ THREE.GLTFExporter.prototype = {
 			var skeleton = object.skeleton;
 			var rootJoint = object.skeleton.bones[ 0 ];
```
Comment (mrdoob) [inline examples/js/exporters/GLTFExporter.js]: Wouldn't this line wail if `skeleton` was `undefined`?

Comment (soccer1mt) [reply examples/js/exporters/GLTFExporter.js]: I get  TypeError: Cannot read property 'bones' of undefined
    at processSkin (GLTFExporter.js:1649)
    at processInput (GLTFExporter.js:1993)
    at ./node_modules/three/examples/js/exporters/GLTFExporter.js.THREE.GLTFExporter.parse (GLTFExporter.js:2005)
    at _callee2$ (ModelBuilder.js:255)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at tryCatch (runtime.js:45)
    at invoke (runtime.js:135)
    at runtime.js:145

Comment (Mugen87) [reply examples/js/exporters/GLTFExporter.js]: The patch needs to be implemented differently. Maybe like so:
```js
var skeleton = object.skeleton;

if ( skeleton === undefined ) return null;

var rootJoint = skeleton.bones[ 0 ];

if ( rootJoint === undefined ) return null;
```

Comment (soccer1mt) [reply examples/js/exporters/GLTFExporter.js]: agreed.  submitting new PR

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**